setup.py

```diff
@@ -91,7 +91,7 @@ def _single_compile(obj):
 # libraries += [current_python]
 
 libraries = []
-include_dirs = ['src', '.','python', 'third_party/eigen3', 'third_party/tinyxml2/include', 'third_party/pybind11/include']
+include_dirs = ['src', '.','python', 'third_party/eigen3', 'third_party/tinyxml2/include', 'third_party/pybind11/include', 'third_party/CppAD/include', 'build/CppADCodeGen/include']
```

Comment:
If you add CppAD/CppADCodeGen to setup.py, don't you need to add tons of other dependencies as well, to make it compile?

Reply:
I regularly compile with `python3 setup.py install` and it seems fine. Just in case I moved the inclusion for just the `pytinydiffsim_ad` extension. See 2726923.